websocket/pom.xml

```diff
@@ -84,12 +82,12 @@
             <build>
                 <plugins>
                     <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Isn't the implicit groupId good enough?

Reply:
I prefer explicit usage of gId

Reply:
you are already changing the area, improve for the sake of DRY